architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,44 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with {product-title} 3.11, you can specify SCCs as a resource that
+is handled by RBAC. This allows you to scope access to your SCCs to a certain
+project or to the entire cluster. Assigning users or groups directly to an
```

Comment:
Maybe,we need add service accounts here,thanks

Reply:
@stlaz, if I'm interpreting @barleyer correctly, please s/Assigning users or groups/ Assigning users, groups, or service accounts

Reply:
Thank you both, I updated the paragraph accordingly.